spec/factories/users.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :user do
+    first_name { Faker::Name.first_name }
+    last_name { Faker::Name.last_name }
+    email { Faker::Internet.email }
```

Comment:
pls use sequences for uniq fields, otherwise it'd give unstable behavior in case of multiple users creation

Reply:
for email yes